src/main/java/com/google/api/codegen/ProtocGeneratorMain.java

```diff
@@ -45,7 +46,40 @@
   private static final ArtifactType DEFAULT_ARTIFACT_TYPE = GAPIC_CODE;
 
   public static void main(String[] args) {
-    // TODO(andrealin): Implement. Handle I/O interfacing with protoc.
+    System.err.println("Parsing System.in for CodeGeneratorRequest.");
```

Comment:
Do we want to always output this to System.err?

Reply:
ahh it was a debugging statement i suppose we can remove